src/components/views/rooms/MessageComposerInput.js

```diff
@@ -523,13 +523,27 @@ export default class MessageComposerInput extends React.Component {
         let sendTextFn = this.client.sendTextMessage;
 
         if (contentText.startsWith('/me')) {
```

Comment:
@turt2live see my latest commit if you like it otherwise I will make it even smaller xD